services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -56,6 +56,36 @@ export default class IndexPage extends React.Component<Props, State> {
               zip: Yup.string()
                 .required('Zip Code Is Required')
                 .min(5, 'Zip Codes Contains 5 Digits'),
+              firstName: Yup.string()
+                .required('Your First Name Is Required!')
+                .min(2, 'Your First Name Needs To Be Valid'),
+
+              lastName: Yup.string()
+                .required('Your Last Name Is Required!')
+                .min(2, 'Your Last Name Needs To Be Valid'),
+              address: Yup.string()
+                .required('Your Address Is Required!')
+                .min(2, 'Your Address Needs To Be Valid'),
+              unit: Yup.number()
+                .required('Your Unit or Apartment Is Required!')
+                .positive()
+                .integer(),
```

Comment:
Shouldn't be required, or forced to be a positive integer ("2B" is a valid apartment number, for example).